github-comment-fetcher/src/main/java/com/tzachz/commentcounter/CommentFetcher.java

```diff
@@ -71,4 +74,19 @@ public void setClock(Clock clock) {
                 .collect(Collectors.toList());
     }
 
+    private List<GHRepo> getInterestingRepos(Set<GHRepo> repos) {
+        final Set<String> repoFilter;
+        if (this.repositories.isEmpty()) {
+            repoFilter = repos.stream()
+                    .map(GHRepo::getName)
+                    .collect(Collectors.toSet());
+        } else {
+            repoFilter = this.repositories;
+        }
+
+        return repos.stream()
+                .peek(repo -> logger.info("repo={}", repo.getName()))
+                .filter(repo -> repoFilter.contains(repo.getName()))
```

Comment:
Another unit test in `CommentFetcherTest` to verify this behavior would be great

Reply:
Wanted to invest time on tests iff you thought the feature useful.  Will add tests too.

Reply:
This is done.  I didn't have a nicer way of mocking the constructor of CommentFetcher in the unit test so I recreated the instance in the unit test.